src/sentry/api/endpoints/organization_events.py

```diff
@@ -126,3 +126,44 @@ def data_fn(offset, limit):
                         request, organization, params["project_id"], results
                     ),
                 )
+
+
+class OrganizationEventsGeoEndpoint(OrganizationEventsV2Endpoint):
+    def get(self, request, organization):
+        if not self.has_feature(organization, request):
+            return Response(status=404)
+
+        try:
+            params = self.get_snuba_params(request, organization)
+        except NoProjects:
+            return Response([])
+
+        fields = request.GET.getlist("field")
+        fields = ["geo.country_code", fields[0]]
+
+        def data_fn(offset, limit):
+            return discover.query(
+                selected_columns=fields,
+                query=request.GET.get("query"),
+                params=params,
+                orderby=self.get_orderby(request),
+                offset=offset,
+                limit=limit,
+                referrer=request.GET.get("referrer", "api.organization-events-geo"),
+                auto_fields=True,
+                auto_aggregations=True,
+                use_aggregate_conditions=True,
```

Comment:
Do we want these to be `True`? Do you see the user making queries like give me the `p75(transaction.duration)` by country and then filtering out countries where the `count():>100`?

Reply:
I removed `auto_fields` and `auto_aggregations`. 

Is it fine if `use_aggregate_conditions` is kept?